src/pages/CommunityResourcesPage/ui/constants.tsx

```diff
@@ -0,0 +1,78 @@
+interface Resource {
```

Comment:
We will have different types of resources, I think they will have different Interface, but let's discuss on Thursday